runtime/src/block_cost_limits.rs

```diff
@@ -40,6 +41,7 @@ lazy_static! {
         (solana_vote_program::id(), COMPUTE_UNIT_TO_US_RATIO * 70),
         // secp256k1 is executed in banking stage, it should cost similar to sigverify
         (secp256k1_program::id(), COMPUTE_UNIT_TO_US_RATIO * 24),
+        (ed25519_program::id(), COMPUTE_UNIT_TO_US_RATIO * 24),
```

Comment:
nit: could you define `SIGNATURE_US = 24` and use it for all of these?: `SIGNATURE_COST`, `secp256k1_program::id()`, `ed25519_program::id()`?